src/components/AppRouter.js

```diff
@@ -40,6 +40,9 @@ const AppRouter = () => {
 					<Route path='/customise' component={CustomiseContainer} />
 					<Route path='/personalised' component={PersonalView} />
 					<Route path='/default' component={Default} />
+					<div className='footer'>
```

Comment:
Can i be annoying? Lets use footer tag for that :)

Reply:
Done :)